app/controllers/admin/categories_controller.rb

```diff
@@ -20,7 +20,13 @@ def new
   end
 
   def edit
-    @category = LinkedData::Client::Models::Category.find_by_acronym(params[:id]).first
+    @category = LinkedData::Client::Models::Category.find_by_acronym(params[:id], include:'name,acronym,created,description,parentCategory,ontologies' ).first
+    @ontologies_category = LinkedData::Client::Models::Ontology.all.map {|o|[o.acronym, o.id] }
+    @id = "category_ontologies"
+    @name = "category[ontologies]"
+    @values = @ontologies_category
+    @selected = @category.ontologies
+    @multiple = true
```

Comment:
you don't need the `@id`, `@name`,  `@values`, `@selected` nad `@multiple`  variable directly put the values in the view;